es/cheatsheet-deep-learning.md

```diff
@@ -1,291 +1,291 @@
 
 **7. where we note w, b, z the weight, bias and output respectively.**
 
-&#10230;
+&#10230; donde w, b y z son el peso, el sesgo y la salida, respectivamente.
 
 <br>
 
 **8. Activation function ― Activation functions are used at the end of a hidden unit to introduce non-linear complexities to the model. Here are the most common ones:**
 
-&#10230;
+&#10230; Función de activación - Las funciones de activación son utilizadas al final de una unidad oculta para introducir complejidades no lineales al modelo. A continuación las más comunes:
 
 <br>
 
 **9. [Sigmoid, Tanh, ReLU, Leaky ReLU]**
 
-&#10230;
+&#10230; [Sigmoide, Tanh, ReLU, ReLU filtrante]
 
 <br>
 
 **10. Cross-entropy loss ― In the context of neural networks, the cross-entropy loss L(z,y) is commonly used and is defined as follows:**
 
-&#10230;
+&#10230; Pérdida de entropía cruzada - En el contexto de las redes neuronales, la pérdida de entropía cruzada L(z,y) es utilizada comunmente y definida de la siguiente manera:
 
 <br>
 
 **11. Learning rate ― The learning rate, often noted α or sometimes η, indicates at which pace the weights get updated. This can be fixed or adaptively changed. The current most popular method is called Adam, which is a method that adapts the learning rate.**
 
-&#10230;
+&#10230; Tasa de aprendizaje - La tasa de aprendizaje, denotada como α o algunas veces η, indica a que ritmo los pesos son actualizados. Este valor puede ser fijo o cambiar de forma adaptativa. El método más popular en este momento es llamado Adam, que es un método que se adapta a la tasa de aprendizaje.
 
 <br>
 
 **12. Backpropagation ― Backpropagation is a method to update the weights in the neural network by taking into account the actual output and the desired output. The derivative with respect to weight w is computed using chain rule and is of the following form:**
 
-&#10230;
+&#10230; Retropropagación - La retropropagación es un método de actualización de los pesos en una red neuronal, tomando en cuenta la salida actual y la salida esperada. La derivada respecto al peso w es computada utilizando la regla de la cadena y se expresa de la siguiente forma:
 
 <br>
 
 **13. As a result, the weight is updated as follows:**
 
-&#10230;
+&#10230; Como resultado, el peso es actualizado de la siguiente forma:
 
 <br>
 
 **14. Updating weights ― In a neural network, weights are updated as follows:**
 
-&#10230;
+&#10230; Actualizando pesos - En una red neuronal, los pesos son actualizados de la siguiente forma:
 
 <br>
 
 **15. Step 1: Take a batch of training data.**
 
-&#10230;
+&#10230; Paso 1: Tomar un grupo de los datos de entrenamiento.
 
 <br>
 
 **16. Step 2: Perform forward propagation to obtain the corresponding loss.**
 
-&#10230;
+&#10230; Paso 2: Realizar prealimentación para obtener la pérdida correspondiente.
 
 <br>
 
 **17. Step 3: Backpropagate the loss to get the gradients.**
 
-&#10230;
+&#10230; Paso 3: Propagar retroactivamente la pérdida para obtener los gradientes.
```

Comment:
To be consisten with item 12 I suggest "backpropagate" to be translated as "Retropropagar"

Reply:
Mhmm. Since we translated _forward propagation_ as _propagación hacia adelante_, it might be a good idea to translate this one as _propagación inversa_. However, the RAE defines the prefix [retro-](http://dle.rae.es/srv/search?m=30&w=retro-) as "backwards" so _retropropagación_ is OK.